src/shared/hooks/useLocalStorage.ts

```diff
@@ -20,6 +20,7 @@ export type UserPreferenceKey =
   | `table columns for ${Namespace}`
   // column selection for the xrefs table views for UniParc entries
   | `table columns for ${Namespace} entry page`
+  | `table columns for ${Namespace} entry page v2`
```

Comment:
I think it would be useful to add some comments here to explain what is happening. Also creating a jira to eventually delete `table columns for ${Namespace} entry page v2` from localStorage and to revert to `table columns for ${Namespace} entry page` could be useful.